assets/js/amp-story-editor-blocks.js

```diff
@@ -342,6 +342,14 @@ var ampStoryEditorBlocks = ( function() { // eslint-disable-line no-unused-vars
 				),
 				el = wp.element.createElement;
 
+			if ( attributes.ampFontFamily ) {
+				newProps.wrapperProps[ 'data-font-family' ] = attributes.ampFontFamily;
+				return el(
+					BlockListBlock,
+					newProps
+				);
+			}
+
```

Comment:
@miina I'm not sure if we need to do some re-factoring here or move this to its own function. As you can see I had it further down in the function before, but that didn't work properly as sometimes it didn't get that far to add the new property.

Reply:
@jacobschweitzer Just made some changes in the code, thanks for noting that.